tests/test_odilo.py

```diff
@@ -0,0 +1,140 @@
+# encoding: utf-8
+from nose.tools import (
+    eq_, ok_
+)
+
+import os
+import json
+
+from odilo import (
+    MockOdiloAPI,
+    OdiloRepresentationExtractor,
+    OdiloBibliographicCoverageProvider
+)
+
+from model import (
+    Contributor,
+    DeliveryMechanism,
+    Edition,
+    Identifier,
+    Representation,
+    Subject,
+    Hyperlink,
+)
+
+from testing import DatabaseTest
+
+
+class OdiloTest(DatabaseTest):
+    def setup(self):
+        super(OdiloTest, self).setup()
+        self.collection = MockOdiloAPI.mock_collection(self._db)
+        base_path = os.path.split(__file__)[0]
+        self.resource_path = os.path.join(base_path, "files", "odilo")
+
+    def sample_json(self, filename):
+        path = os.path.join(self.resource_path, filename)
+        data = open(path).read()
+        return data, json.loads(data)
+
+
+class OdiloTestWithAPI(OdiloTest):
+    """Automatically create a MockOdiloAPI class during setup.
+    """
+
+    def setup(self):
+        super(OdiloTestWithAPI, self).setup()
+        self.api = MockOdiloAPI(self._db, self.collection)
+
+
+class TestOdiloBibliographicCoverageProvider(OdiloTest):
+    def setup(self):
+        super(TestOdiloBibliographicCoverageProvider, self).setup()
+        self.provider = OdiloBibliographicCoverageProvider(
+            self.collection, api_class=MockOdiloAPI
+        )
+        self.api = self.provider.api
+
+    def test_process_item(self):
+        print 'Testing process item...'
```

Comment:
All print statements should be removed or converted into `self.log` statements. This one shouldn't be necessary because making it clear which test is being run is the responsibility of the test infrastructure.

Reply:
Ok, I have either deleted or changed them at both test files.